runtime/integration-tests/src/xcm/development/tests/connectors.rs

```diff
@@ -256,7 +260,7 @@ fn update_token_price() {
 }
 
 #[test]
-fn transfer() {
```

Comment:
Can we add tests for all the new messages? 🙏

Reply:
Of course, sorry for having missed out on these.

Reply:
AFAICS, basically only `collect_redeem` was missing. Added a few more `zero` cases. [7f60bd4](https://github.com/centrifuge/centrifuge-chain/pull/1292/commits/7f60bd4cc51be66013ec756b264762fadad83119)